### ISSUE TITLE
[v1.0-Scenarios-CalendarRemainder.json]: Label 'Snooze for' is not associated with '5 minutes' button in 'CalendarRemainder.json' card when user navigate to '5 minutes' button in backward navigation.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7346]
https://microsoft.visualstudio.com/OS/_workitems/edit/37842122
